operate/client/src/modules/stores/processInstanceListeners.ts

```diff
@@ -175,11 +182,19 @@ class ProcessInstanceListeners {
   fetchPreviousInstances = async () => {
```

Comment:
I think I was a bit divided on the nomenclature when I was working on the Execution Listeners as to if the items listed on this were the actual listeners or instances of a listener, since the Listener itself would be element attached to the task in the process.

I guess it's just semantics, though, and `Listeners` is more descriptive than `Instances` in this scenario, so I'll commit the change along with the fix for the other comment.